File: backend/DiplomskaNaloga/Data/Entity/SensorGroup.cs
﻿using System;
namespace Data.Entity
{
	public partial class SensorGroup
	{
		public Guid Id { get; set; }
		public Guid UserId { get; set; }
		public string Name { get; set; }
		public DateTime CreatedAt { get; set; }
		public DateTime UpdatedAt { get; set; }
		public string? ColumnX { get; set; }
		public string? ColumnY { get; set; }
	}
}

File: frontend/diplomska-naloga/src/app/interfaces/sensor-group.ts
export interface SensorGroup {
    id:        string;
    userId:    string;
    createdAt: Date;
    updatedAt: Date;
    name:      string;
    sensorHash: string;
    showHash:boolean;
    description: string;
    lat: number;
    long: number;
}
